canairio/settings/base.py

```diff
@@ -124,43 +110,17 @@
 
 TRACK_COLLECTION_NAME = 'test_tracks_data'
 
-FB_API_KEY = os.getenv('FB_API_KEY', '')
-FB_AUTH_DOMAIN = os.getenv('FB_AUTH_DOMAIN', '')
-FB_DATABASE_URL = os.getenv('FB_DATABASE_URL', '')
-FB_STORAGE_BUCKET = os.getenv('FB_STORAGE_BUCKET', '')
+# Firebase
+FB_API_KEY = ''
+FB_AUTH_DOMAIN = ''
+FB_DATABASE_URL = ''
+FB_STORAGE_BUCKET = ''
 
-INFLUXDB_HOST = os.getenv('INFLUXDB_HOST', '')
+INFLUXDB_HOST = getenvvar('INFLUXDB_HOST')
 INFLUXDB_PORT = 8086
 INFLUXDB_USERNAME = None
 INFLUXDB_PASSWORD = None
 INFLUXDB_DATABASE = 'canairio'
 INFLUXDB_TIMEOUT = 10
 
-ENVIRONMENT = os.getenv('ENVIRONMENT', ENVIRONMENT)
-if 'production' in ENVIRONMENT:
-
-
-    FB_API_KEY = os.getenv('FB_API_KEY', '')
-    FB_AUTH_DOMAIN = os.getenv('FB_AUTH_DOMAIN', '')
-    FB_DATABASE_URL = os.getenv('FB_DATABASE_URL', '')
-    FB_STORAGE_BUCKET = os.getenv('FB_STORAGE_BUCKET', '')
-
-    INFLUXDB_HOST = os.getenv('INFLUXDB_HOST', '')
-    INFLUXDB_PORT = 8086
-    INFLUXDB_USERNAME = None
-    INFLUXDB_PASSWORD = None
-    INFLUXDB_DATABASE = os.getenv('INFLUXDB_DATABASE', '')
-    INFLUXDB_TIMEOUT = 10
-
-    DATABASES = {
-        'default': config(
-            default=config('DATABASE_URL')
-        )
-    }
-else:
-    try:
-        from canairio.settings_local import *
-    except ImportError:
-        print('You must create a settings_local.py file.')
-
```

Comment:
We should be instrumenting `settings.py` to be able to manage different environment configurations.

Reply:
The overall proposal would be to have a single settings.py and work everything with envvars?  We started to run out of envvars while using Amazon with elasticbeanstalk.

Reply:
That's only the case if we introduce many dependencies and we're not at that point. We should only introduce complexity when we actually need it. At this point this is a YAGNI.

Reply:
Also, take a look at this if you're still facing the problem of running out of envvars, it's about your data structure size which can be changed:

https://docs.aws.amazon.com/opsworks/latest/userguide/workingapps-creating.html#workingapps-creating-environment

Reply:
As a followup, if we actually ever reach that state, i.e. if our envvars are over 4096 in length and or over the allowed limit by the infrastructure provider, we would instead switch to a JSON config scheme or a Chef recipe configuration setting. The general idea is that we would then be completely decoupled from our infrastructure when dealing with configuration changes.

Reply:
Here we are only missing the one line configuration for the database.  Can you give me a hand on this one?